plasmapy/formulary/dimensionless.py

```diff
@@ -171,11 +171,10 @@ def Mag_Reynolds(U: u.m / u.s, L: u.m, sigma: u.S / u.m) -> u.dimensionless_unsc
 
     .. math::
 
-        Rm = \frac{U L}{\\eta}
+        Rm = \frac{U L}{\eta}
 
-        where: \eta = \frac{1}{\mu_0 \sigma}
-
-        and \mu_0 is the permeability of free space.
+    where :math:`\eta = \frac{1}{\mu_0 \sigma}`
+    and :math:`\mu_0` is the permeability of free space.
```

Comment:
It definitely looks like something is off with your git tree history... These changes should really not be showing up. Could you try merging the current master branch?